ibc-testkit/src/testapp/ibc/clients/mock/consensus_state.rs

```diff
@@ -9,11 +9,17 @@ use crate::testapp::ibc::clients::mock::header::MockHeader;
 use crate::testapp::ibc::clients::mock::proto::ConsensusState as RawMockConsensusState;
 pub const MOCK_CONSENSUS_STATE_TYPE_URL: &str = "/ibc.mock.ConsensusState";
 
+// Note: This type differs from the `mock::ConsensusState` type exposed by
+// ibc-proto in a few ways:
+// - ibc-proto's `mock::ConsensusState`'s `header` field takes the form of an
+//   `Option<Header>`, while the `header` field here is not optional.
+// - ibc-proto's `mock::ConsensusState` does not have a `root` field for holding
+//   the `CommitmentRoot`; this field exists here because ...
```

Comment:
Can you make it a Rust doc-comment?

Also, no need to comment on the `Option<Header>` as that's an issue from [protobuf](https://github.com/protocolbuffers/protobuf/issues/2497) itself.

If you may, you can comment on why we _had_ to keep a `root` field and not return a constant at `ConsensusState::root` implementation.

Reply:
I have to demonstrate to you. So here is an example.

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=9aadf23ade52af889d2026193a70a127